Tests/TestUtilities/TestUtils.cs

```diff
@@ -156,12 +156,35 @@ public static string CreateValidEmptyRuleset(string parentDir, string fileNameWi
     /// Creates a batch file with the name of the current test
     /// </summary>
     /// <returns>Returns the full file name of the new file</returns>
-    public static string WriteBatchFileForTest(TestContext context, string content)
+    public static string WriteExecutableScriptForTest(TestContext context, string content)
     {
+        var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+
+        var fileExtension = isWindows ? ".bat" : ".sh";
         var testPath = CreateTestSpecificFolder(context);
-        var fileName = Path.Combine(testPath, context.TestName + ".bat");
-        File.Exists(fileName).Should().BeFalse("Not expecting a batch file to already exist: {0}", fileName);
-        File.WriteAllText(fileName, content);
+        var fileName = Path.Combine(testPath, context.TestName + fileExtension);
+        File.Exists(fileName).Should().BeFalse("Not expecting a script file to already exist: {0}", fileName);
+        File.WriteAllText(fileName, isWindows ? content : content.NormalizeLineEndings());
```

Comment:
Here it should be possible to simplify to 
```suggestion
        File.WriteAllText(fileName, content.NormalizeLineEndings());
```
right?